com.unity.render-pipelines.high-definition/Editor/RenderPipeline/PathTracing/PathTracingEditor.cs

```diff
@@ -57,6 +66,35 @@ public override void OnInspectorGUI()
                         PropertyField(m_MaxDepth);
                         PropertyField(m_MaxIntensity);
                         PropertyField(m_SkyImportanceSampling);
+#if ENABLE_UNITY_DENOISING_PLUGIN
+                        PropertyField(m_Denoising);
+                        var denoiserType = m_Denoising.value.GetEnumValue<DenoiserType>();
+                        bool supported = Denoiser.IsDenoiserTypeSupported(denoiserType);
+
+                        if (m_Denoising.value.intValue != (int) DenoiserType.None)
+                        {
+                            using (new IndentLevelScope())
+                            {
+                                if (supported)
+                                {
+                                    PropertyField(m_UseAOV);
+                                    if (m_Denoising.value.intValue == (int)DenoiserType.Optix)
+                                    {
+                                        PropertyField(m_Temporal);
+                                    }
+                                }
+                                else
+                                {
+                                    EditorGUILayout.HelpBox($"The selected denoiser is not supported by this hardware configuration.", MessageType.Error, wide: true);
+                                }
+                            }
+                        }
+#else
+                        CoreEditorUtils.DrawFixMeBox("Path Tracing Denoising is not active in this project. To activate it, install the Unity Denoising Plugin package.", MessageType.Info, () =>
```

Comment:
Unity Denoising Plugin or Unity Denoiser Plugin?
I've seen both in this OR. 
The first of these two options sounds more grammatical to me.

Reply:
I think I have Denoising in all publicly visible places right now. Before release we have the flexibility to change that. 
In the scripting API, we do use "Denoiser" for the class that performs the denoising.